### PR TITLE
Fix NO_NAVSTART error for file: uri

### PR DESCRIPTION
**Pull request by mbalabash**
It will fix problem according to https://github.com/aslushnikov/tracium/issues/3

### CODE REVIEW

File: lib/trace-of-tab.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const log = require('debug')(`tracium`);
 const TracingProcessor = require('./tracing-processor.js');
 const LHError = require('./lh-error.js');
 
-const ACCEPTABLE_NAVIGATION_URL_REGEX = /^(chrome|https?):/;
+const ACCEPTABLE_NAVIGATION_URL_REGEX = /^(chrome|https|file?):/;
```
Comment (patrickhulce) [inline lib/trace-of-tab.js]: This should be `/^(chrome|https?|file):/`

the `?` is for the optional `s` on `https` :)

**Review by patrickhulce — CHANGES_REQUESTED** [review]
Thanks for the quick fix @mbalabash!

File: lib/trace-of-tab.js (hunk 1 above)
Comment (mbalabash) [reply lib/trace-of-tab.js]: Yep)
You right. My mistake.

Comment (mbalabash) [reply lib/trace-of-tab.js]: I updated code.

**Comment by paulirish** [comment]
published in 0.1.1

**Comment by mbalabash** [comment]
Thanks

**Pull request merged**